examples/ModelDriftAnalysis/start_torchserve.py

```diff
@@ -0,0 +1,46 @@
+import atexit
+import os
+import shutil
+import subprocess
+import json
+import time
+
+
+def start_torchserve():
```

Comment:
All these methods are avaialble in https://github.com/chauhang/mlflow-torchserve-1/blob/master/tests/fixtures/plugin_fixtures.py . Check if it can be reused